test/usbmux/usbmux-specs.js

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
```

Comment:
`import { fs } from 'appium-support';` and then, instead of `fs.readFileSync` use `await fs.readFile`.